### ISSUE TITLE
Output report as markdown

### ISSUE THREAD

**Issue by hankanman** [dummylabs/thewatchman#10]
Can you add an option to output the report as markdown? makes it nice and readable in vscode as a preview and also solves issue #3 as you can just render the file using a markdown card :). Love the integration btw

**Comment by dummylabs** [comment]
Yes, that makes sense, along with other output options like CSV.
As for #3, this is already implemented and will be a part of the next release. The approach is slightly different: each watchman sensor will have a data structure as an attribute, which you can use with a template to render simply anything.
